packages/component-library/src/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
```

Comment:
It looks like this file is not used and should be deleted.

Reply:
This references the entire file.

Reply:
To clarify -- `RadioButton.js` is unused, because it was superseded by `RadioButtonGroup.js`

Reply:
When I remove this line the eslint message:
'React' must be in scope when using JSX.

Reply:
Understood, the file  RadioButton.js should be deleted.